docs/plugins/export.rst

```diff
@@ -36,24 +39,36 @@ The ``export`` command has these command-line options:
 
 * ``--append``: Appends the data to the file instead of writing.
 
+* ``--format`` or ``-f``: Specifies the format the data will be exported as. If not informed, JSON will be used by default. The format options include csv, json and xml.
+
 Configuration
 -------------
 
 To configure the plugin, make a ``export:`` section in your configuration
-file. Under the ``json`` key, these options are available:
+file. Under the ``json``, ``csv``, and ``xml`` keys, these options are available:
 
-- **ensure_ascii**: Escape non-ASCII characters with ``\uXXXX`` entities.
+- **JSON Formatting**
+    - **ensure_ascii**: Escape non-ASCII characters with ``\uXXXX`` entities.
 
-- **indent**: The number of spaces for indentation.
+    - **indent**: The number of spaces for indentation.
 
-- **separators**: A ``[item_separator, dict_separator]`` tuple.
+    - **separators**: A ``[item_separator, dict_separator]`` tuple.
 
-- **sort_keys**: Sorts the keys in JSON dictionaries.
+    - **sort_keys**: Sorts the keys in JSON dictionaries.
 
 These options match the options from the `Python json module`_.
 
 .. _Python json module: https://docs.python.org/2/library/json.html#basic-usage
 
+- **CSV Formatting**
+    - **delimiter**: Used as the separating character between fields. The default value is a comma (,).
+
+    - **dialect**: A dialect is a construct that allows you to create, store, and re-use various formatting parameters for your data.
```

Comment:
```suggestion
    - **dialect**: The kind of CSV file to produce. The default is `excel`.
```

Keeping this focused on what matters to *users of this plugin*, not to Python programmers.